_docs/_latest/use/model.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Model
+order: 004
+---
+
+# {{ page.title }}
+
+Forseti Model is built on top of the raw inventory data to provide us with a better 
```

Comment:
s/Model/models/.

`..raw inventory data to build a representation of the effective policy in total.`

Reply:
Updated